commitlint.config.ts

```diff
@@ -228,6 +228,18 @@ module.exports = {
                     ];
                 },
 
+                'empty-wip': ({header}: {header:any}) => {
```

Comment:
@parhamsaremi same here, move up because of alphabetical order pls

Reply:
I moved this up but I think other rules have this order problem too :thinking: